media/src/activity/activity-map-pane.tsx

```diff
@@ -225,7 +225,12 @@ export const ActivityMapPane = React.forwardRef<HTMLDivElement, ActivityMapPaneP
                             {
                                 handler: deleteProject,
                                 icon: <FontAwesomeIcon icon={faTrashAlt}/>,
-                                label: 'Delete project'
+                                label: 'Delete project',
+                                confirmationTitle: 'Delete project?',
+                                confirmationText:
+                                    'Are you sure that you want to delete this project?' +
```

Comment:
There's a space character missing here, between the question mark and the next word.

Reply:
Thanks!